examples/zenohpico/z_get.c

```diff
@@ -0,0 +1,119 @@
+//
```

Comment:
These are Zenoh-Pico examples.
Why do we have them in Zenoh-CPP as well?

Reply:
This is for historical reasons. I had to test compilation zenohc (and later zenohpico) library with C++ compiler. So I copied examples to zenoh-cpp to test C++ compilation of all zenohc functionality. Then I followed the same logic for zenohpico. Maybe now, when C++ library covers the whole API, we can simply remove these examples to not confuse user.

Reply:
Done